tools/targets/ARM_MUSCA_S1.py

```diff
@@ -109,3 +140,12 @@ def find_bl2_size(configFile):
                 bl2_size = int(m.group(1), 0)
                 break
     return bl2_size
+
+def run_cmd(cmd, directory):
+
+    POPEN_INSTANCE = subprocess.Popen(
+        cmd,
+        cwd=directory,
+    )
+
+    POPEN_INSTANCE.communicate()
```

Comment:
`run_cmd` assumes that signing will always succeed. It should check at least `returncode` to see if process succeeded. If failed then tap into `stderr` and print it on the console, if not then failure information will be lost.